src/str.js

```diff
@@ -700,6 +701,46 @@ Sk.builtin.str.prototype["replace"] = new Sk.builtin.func(function (self, oldS,
     return new Sk.builtin.str(self.v.replace(patt, replacer));
 });
 
+Sk.builtin.str.prototype["zfill"] = new Sk.builtin.func(function (self, len) {
```

Comment:
I am a little curious about the complexity of the zfill implementation.  After reading over the python docs for this method I am wondering if there is a simpler/clearer solution?

```js
// figure out how many zeroes are needed to make the proper length
var zeroes = len - str.length;
// offset by 1 if there is a +/- at the beginning of the string
var offset = (str[0] === "+" || str[0] === "-") ? 1 : 0;
var pad = "";
// exit early if no zeroes are needed
if (zeroes <= 0) return str;
// create a string of as many zeroes as are needed
while(zeroes--) pad += "0";
// combine the string and the zeroes
return str.substr(0, offset) + pad + str.substr(offset);
```